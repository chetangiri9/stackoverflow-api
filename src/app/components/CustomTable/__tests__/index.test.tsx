import * as React from 'react';
import { render } from '@testing-library/react';

import { CustomTable } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<CustomTable  />', () => {
  const data = [{ title: '', body: '', link: '' }];
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <CustomTable data={data} handleRowClick={() => jest.fn()} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
