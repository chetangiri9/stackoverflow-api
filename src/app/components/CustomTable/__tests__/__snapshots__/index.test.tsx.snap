// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomTable  /> should match snapshot 1`] = `
<div
  style="overflow: auto; display: flex; text-align: center;"
>
  <table>
    <thead>
      <tr>
        <th
          style="width: 0px;"
        >
          Author
        </th>
        <th
          style="width: 0px;"
        >
          Title
        </th>
        <th
          style="width: 0px;"
        >
          Creation Date
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td />
        <td
          style="text-align: left; height: 100px;"
        />
        <td />
      </tr>
    </tbody>
  </table>
</div>
`;
